nflow-engine/src/main/resources/nflow-engine.properties

```diff
@@ -65,5 +65,6 @@ nflow.db.max_pool_size=4
 nflow.db.idle_timeout_seconds=600
 nflow.db.create_on_startup=true
 nflow.db.disable_batch_updates=false
+nflow.db.type.cache.size=10000
```

Comment:
Add nflow.executor.fetchChildWorkflowIds with default value here as well, and make it required in code